lib/helpers/isRouteWithFallback.js

```diff
@@ -0,0 +1,9 @@
+const getPrerenderManifest = require("./getPrerenderManifest");
+
+const { dynamicRoutes } = getPrerenderManifest();
+
+const isRouteWithFallback = (route) => {
+  return route && dynamicRoutes[route].fallback;
```

Comment:
```suggestion
  return dynamicRoutes[route] && dynamicRoutes[route].fallback;
```

`dynamicRoutes[null]` is `undefined`, so no need to check for presence of `route &&` :)

Reply:
Done ✅